Tutorial-1.md

```diff
@@ -0,0 +1,115 @@
+# Tutorial: Making a Simple UI Change
+
+## Introduction
+
+This tutorial guides you through making a simple UI change on the Oppia website. Specifically, we'll update the text on the top of the About page from "Get Started with Oppia" to "Introducing Oppia."
+
+## Skills Covered
+
+1. Figuring out which file(s) in the codebase to modify.
+2. Understanding how to work with internationalization (i18n) keys.
+3. Manually testing changes in both English and another language.
+
+## Scenario
+
+The UX writing team has filed a request on GitHub asking to update the text on the top of the About page. Let's proceed step by step.
+
+## Procedure
+
+<i>The following steps illustrate how a developer might tackle this issue. Try following this tutorial step-by-step on your local machine! This will give you a better sense of how to tackle other similar issues in the codebase.
+
+**Important:**
+When you see a “question box”, stop and try to figure out the answer on your own before reading ahead. You will learn more if you try to figure out your own answer to the question first!</i>
+
+### Stage 1: Figure out the root cause of the issue and which files are affected.
+
+**Setup:**
+Start your local Oppia server. If you haven’t installed Oppia on your local machine, please follow the steps in [this wiki page](https://github.com/oppia/oppia/wiki/Installing-Oppia).
+
+**Identify the Element:**
+When the server is started, it navigates to the home page by default. Since the changes have to be done in the About page, let’s head to the About page at localhost:8181/about.
+
+<img src="images/tutorial-1/HomePage.png" alt="Practice 1 Image">
+
+> [!NOTE]
```

Comment:
Hi @seanlip FYI, for this line when I review changes in file tab I get something like this  
<img width="1440" alt="Screenshot 2024-01-21 at 20 23 33" src="https://github.com/oppia/oppia-web-developer-docs/assets/98253080/9902c422-f62c-4236-afd4-58e820df25a1">
 But when i go to edit mode in file tab I see something like this in preview

<img width="1440" alt="Screenshot 2024-01-21 at 20 24 11" src="https://github.com/oppia/oppia-web-developer-docs/assets/98253080/e818a8a0-6f18-4af1-b1d0-bc8b660b9d8e">

Reply:
I think it's probably fine, my only concern is "Note" which indicates something skippable. Are there any other options we can use that are more along the lines of "Do this" or "Attention"?

Reply:
Okay, I saw one labelled as 'Attention' in documentation so I have replaced Note with it. Thanks.